ros/src/simulation/gazebo_simulator/README.md

```diff
@@ -0,0 +1,53 @@
+# Overview
+Autonomous driving car simulator based on gazebo.
+The following sensors are simulated.
+- velodyne (VLP-16, HDL-32E)
+- camera
+- imu
+![screenshot from 2019-01-11 00-42-10-min](https://user-images.githubusercontent.com/8327598/50984661-bed3f780-1545-11e9-9af6-071ddd1cec76.png)
+
```

Comment:
Please describe the architecture around this gazebo simurator by `rqt_graph` or PPT figure?

Reply:
d0893866728dbb48644cb3ac5eb89f7df6875821